svf-llvm/include/SVF-LLVM/SVFIRExtAPI.h

```diff
@@ -0,0 +1,90 @@
+//===- SVFIRExtAPI.h -- Exteral function IR of SVF ---------------------------------------------//
+//
+//                     SVF: Static Value-Flow Analysis
+//
+// Copyright (C) <2013->  <Yulei Sui>
+//
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//===----------------------------------------------------------------------===//
+
+/*
+ * SVFIRExtAPI.h
+ *
+ *  Created on: 18, 5, 2023
+ *      Author: Shuangxiang Kan
+ */
+
+#ifndef INCLUDE_SVFIREXTAPI_H_
+#define INCLUDE_SVFIREXTAPI_H_
+
+#include "Util/ExtAPI.h"
+#include "SVF-LLVM/SVFIRBuilder.h"
+
+namespace SVF
+{
+
+class SVFIRExtAPI
```

Comment:
We don't need to create a new class and you can keep all the function declarations in SVFIRBuilder.cpp and put the function bodies in SVFIRExtAPI.cpp.

Reply:
I'll delete the SVFIRExtAPI.h and put all function bodies in SVFIRExtAPI.cpp.